src/Uno.Extensions.Toolkit.UI/WindowExtensions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Uno.Extensions;
+
+public static class WindowExtensions
+{
+	public static IThemeService GetThemeService(this Window window, ILogger? logger = default) =>
+		new ThemeService(()=>window.Content.XamlRoot, new Dispatcher(window), logger);
```

Comment:
```suggestion
		new ThemeService(() => window.Content?.XamlRoot, new Dispatcher(window), logger);
```
this should have a null guard